Liroy/check_ncdu.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#This script checks whether ncdu is istalled. If not, it asks the user if they want to install it.
+
+which ncdu &> /dev/null
+if [[ $? -eq 0 ]]; then
+  echo "ncdu is installed. Goodbye"
+  exit 0
+fi
+
+read -p "ncdu is not installed on your machine. Would you like to install it? [y/n]" user_answer
```

Comment:
Stopping a script to ask something is a broken workflow. There are other ways to let the user specify what he wants.

Reply:
Do you mean to command line arguments?

Reply:
@lioramilbaum I think that Liroy went by the book, as the Challenge stated:

> Advanced : Write a script that checks if ncdu is installed. If not, ask the user if he wants to install it.

Reply:
I can see it now.  Something for us to improve :-)